build.sh

```diff
@@ -28,7 +28,7 @@
 
 REV=0.9.2
 ZIP_FILENAME=WiFi101-Updater-ArduinoIDE-Plugin-$REV
-REQUIRED_JARS="pde.jar arduino-core.jar jssc-2.8.0-arduino1.jar bcpg-jdk15on-152.jar bcprov-jdk15on-152.jar"
+REQUIRED_JARS="pde.jar arduino-core.jar jssc-2.8.0-arduino1.jar bcpg-jdk15on-152.jar bcprov-jdk15on-152.jar commons-lang3-3.3.2.jar"
```

Comment:
Unrelated to these changes, but to use with IDE 1.8.7 `jssc-2.8.0-arduino1.jar` needs to be changed to `jssc-2.8.0-arduino2.jar`

Reply:
@cmaglie @facchinm @sandeepmistry  could add for now both the .jar in ` REQUIRED_JARS` like
```REQUIRED_JARS="pde.jar arduino-core.jar jssc-2.8.0-arduino1.jar jssc-2.8.0-arduino2.jar bcpg-jdk15on-152.jar bcprov-jdk15on-152.jar commons-lang3-3.3.2.jar"```
 what do you think?

Reply:
does it compile? BTW I don't see the problem in keeping just the latest...

Reply:
with the ide 1.8.7 and using only jssc-2.8.0-arduino2.jar it compile